### ISSUE TITLE
[FR] Allow manually enter a date without using the date picker

### ISSUE THREAD

**Issue by annieappflowy** [AppFlowy-IO/AppFlowy#2441]
### Description

There isn't a way to quickly enter a date that's not around the current year :(
For example, entering birth years would not be easy-breezy

### Impact

Entering dates using the calendar picker

**Comment by hmncube** [comment]
@annieappflowy Are there any designs on how this should look like?

**Comment by annieappflowy** [comment]
Yes, as @Xazin said, we can build a simple but functional one first.
I don't have a design cc: @hmncube

**Comment by MacMeikel** [comment]
Some suggestions for improvement for entering the date in the calendar picker. 1a) Entering the date using the calendar picker is very practical if the date is close to the current date. If the date is in the next year, you have to click several times to reach the target month. For this reason, it should be possible to select the month and year separately (see screenshot 1).  1b) Another realization would be the selection of the year from a pull-down list, within which you can scroll (see screenshot 2). 1c) Another interesting realization can be seen in the video, where you can switch between months and years without additional buttons (I don't know how to describe this 😁.) 2) It would be practical if the date could also be entered directly via a text field (s. screenshot 3).
![grafik](https://github.com/AppFlowy-IO/AppFlowy/assets/150600032/2769db0c-d552-489a-9c33-985ff0e917f4)
![grafik](https://github.com/AppFlowy-IO/AppFlowy/assets/150600032/701be4a2-9bba-4a30-9cbd-4a67971cff54)
![Appflowy_Datumsauswahl](https://github.com/AppFlowy-IO/AppFlowy/assets/150600032/54c81bf9-3d99-4a9e-b769-814b567a3a0b)
https://github.com/AppFlowy-IO/AppFlowy/assets/150600032/c0e43cf3-00f5-4591-8d66-8f235a0ed3c4
